lti/sentry.py

```diff
@@ -0,0 +1,19 @@
+"""Report exceptions to Sentry."""
+
+import os
+
+import raven
+
+
+def get_raven_client(request):
+    """Return the Raven client for reporting crashes to Sentry."""
+    client = request.registry["raven.client"]
```

Comment:
I've added just the IP address for now. The LTI app doesn't have any user context (such as userids or anything) of its own. It _does_ receive some context about the user from Canvas for some requests but I'm not sure yet exactly what we get from Canvas and when, not sure what we'd want to add.